css/components/Image/image.css

```diff
@@ -0,0 +1,22 @@
+.img {
+  box-shadow: 0.1rem 0.1rem 0.2rem 0.1rem var(--COLOR-SLATE-400);
+  height: 12rem;
+  width: 12rem;
+}
+
+.img-container {
+  max-width: 75rem;
```

Comment:
You could use % or vw with width. I remember Sudesh mentioning this thing to me during one of my code review.

Reply:
Thanks for pointing this out, I will look into it.